strawberry_django/utils/typing.py

```diff
@@ -62,27 +82,19 @@ def get_django_definition(
     )
 
 
-def is_auto(obj: Any) -> TypeGuard[StrawberryAuto]:
+def is_auto(obj: Any) -> bool:
     if isinstance(obj, str):
-        # Support future references
         return obj in ["auto", "strawberry.auto"]
 
     return isinstance(obj, StrawberryAuto)
 
 
-def fields(obj: WithStrawberryObjectDefinition) -> list[StrawberryField]:
-    return obj.__strawberry_definition__.fields
```

Comment:
@bellini666 can you tell me about why this was removed?  our team was using this and i am failing to see where this method may have moved to during this refactor.

Reply:
thanks, i'll prefer the latter since using "d-under" fields in libraries is typically frownded upon.